stdlib/public/SDK/Foundation/IndexPath.swift

```diff
@@ -156,7 +156,7 @@ public struct IndexPath : ReferenceConvertible, Equatable, Hashable, MutableColl
             }
         }
         
-        mutating func append(contentsOf other: [Int]) {
+        mutating func append(contentsOf other: __owned [Int]) {
```

Comment:
this probably isn't necessary since it's just `[Int]`

Reply:
No, Array of Int is not a trivial type, i.e. involves reference counting of the array buffer.

Reply:
Right, but IndexPath doesn't gain anything by taking the array owned, since it's not going to be able to use that buffer for storage, and the type isn't generic.

Reply:
You are right, assuming that the case where self is empty and size of other > 2 is not the common case, it's better to pass other as guaranteed.
(I misread airspeedswift's comment: I though he meant that it's not worth annotating because of the parameter type).